### PR TITLE
14145 fix incorrect pdf names update checkbox for ccc and add bc limited pdf

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#14145

*Description of changes:*
 - App version 4.3.16
 - Remove extra underscore from BEN, CCC, and ULC sample incorporation pdf's names
 - Add placeholder pdf for BC Limited corps
 - CCC only have the option for a custom incorporation agreement
 - Update unit test for CCC Incorporation step

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: tests/unit/AgreementType.spec.ts [hunk 1]
```diff
@@ -37,34 +37,30 @@ const agreementTypeTestCases = [
     'has been completed and a copy has been added to the company\'s record book.',
     customSummaryText: 'The custom Incorporation Agreement and custom Articles ' +
     'has been completed and a copy has been added to the company\'s record book.'
+  }
```
Comment (lambert-alex) [inline tests/unit/AgreementType.spec.ts:40]: It looks like there is a lot of changes in this file but the only change is the CCC test cases are now tested the same as the ULC to reflect CCC's no longer having a sample summary. So it is just some rearranging no new tests were added or deleted.

File: src/resources/Incorporations/limitedCompany.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export const LimitedCompanyResource: IncorporationResourceIF = {
         ]
       }
     ],
-    article: 'Sample_articles.pdf',
+    article: 'bc_limited_company_corporation_agreement.pdf',
```
Comment (lambert-alex) [inline src/resources/Incorporations/limitedCompany.ts:83]: Added as a place holder until the new pdf is created. Requested by UX assurance.

File: src/resources/Incorporations/communityContributionCompany.ts [hunk 3]
```diff
@@ -75,14 +75,8 @@ export const CommunityContributionCompanyResource: IncorporationResourceIF = {
...
       }
     ],
-    article: 'community_contribution_company__corporation_agreement.pdf',
+    article: 'community_contribution_company_corporation_agreement.pdf',
     documents: [
-      {
-        code: 'sample',
-        description: 'The <b>sample Incorporation Agreement and Articles</b> containing the community provision' +
-        ' required by the Business Corporations Act 51.911 has been completed and a copy has been added to the' +
-        ' company\'s record book.'
```
Comment (lambert-alex) [inline src/resources/Incorporations/communityContributionCompany.ts:84]: CCC's do not have a sample agreement. Removed at the request of UX

File: src/components/common/AgreementType.vue [hunk 4]
```diff
@@ -22,7 +22,7 @@
     </template>
 
     <!-- EDIT SECTION -->
-    <div v-if="isTypeBcUlcCompany" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
+    <div v-if="isTypeBcUlcCompany || isTypeBcCcc" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
```
Comment (lambert-alex) [inline src/components/common/AgreementType.vue:25]: To reflect that CCC's do not have a sample agreement the they now use the checkbox style agreement confirmation.

Screenshot 2022-11-25 114830

Comment (severinbeauvais) [reply src/components/common/AgreementType.vue:25]: Sounds good. Thanks for the screenshot 👍

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**